filament/src/PostProcessManager.cpp

```diff
@@ -183,4 +185,176 @@ void PostProcessManager::finish(driver::TargetBufferFlags discarded,
     commands.clear();
 }
 
+
+// ------------------------------------------------------------------------------------------------
+
+FrameGraphResource PostProcessManager::msaa(FrameGraph& fg,
```

Comment:
Maybe rename these methods to createMsaaPass, createToneMappingPass, createFxaaPass, createDynamicScalingPass?